pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -77,10 +77,10 @@ section.bg-pattern-dark {
 
 // Adds 2rem of space above or below an element
 .add-margin-top {
-  margin-top: 2rem;
+  margin-top: 2rem !important;
```

Comment:
can you add a comment as to why `!important` is needed here?

Also, more broadly, sorry for being a stickler here -- as you know, this is an anti-pattern and I'd like to understand why we need to do this. I completely believe that it's needed, just want to know why :)

Reply:
Yes definitely! I'll do it in a followup, but the short answer is that this can be used as a utility class that will override existing margin-top on any element.